### PR TITLE
Add Flow definitions to atRule utils

### PR DESCRIPTION
**Pull request by m-allanson**
> Which issue, if any, is this issue related to?

#1974 

> Is there anything in the PR that needs further explanation?

This is a very small PR to confirm that I'm doing the right thing.

### CODE REVIEW

File: src/utils/atRuleParamIndex.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
+/* @flow */
 /**
  * Get the index of a media query's params
  *
  * @param {AtRule} atRule
  * @return {int} The index
  */
-export default function (atRule) {
+export default function (atRule: postcss$atRule) {
```
Comment (davidtheclark) [inline src/utils/atRuleParamIndex.js]: It would also be good to specify the return values on these functions.

Comment (m-allanson) [inline src/utils/atRuleParamIndex.js]: Done

Comment (m-allanson) [inline src/utils/atRuleParamIndex.js]: Ah that makes sense, thanks. I'll get this updated.

Comment (m-allanson) [inline src/utils/atRuleParamIndex.js]: This is updated and rebased now

**Comment by davidtheclark** [comment]
👍

**Pull request merged**